flutter/lib/src/default_integrations.dart

```diff
@@ -335,6 +335,9 @@ class NativeSdkIntegration extends Integration<SentryFlutterOptions> {
   @override
   FutureOr<void> call(Hub hub, SentryFlutterOptions options) async {
     _options = options;
+    if (!options.autoInitializeNative) {
```

Comment:
```suggestion
    if (!_options.autoInitializeNative) {
```

Reply:
The property `_options` is optional (🥁 ), therefore i used the param, which is the same instance.